x/qgb/keeper/query_data_commitment.go

```diff
@@ -8,6 +8,7 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
+// FIXME should we return an error if it doesn't exist? for this one and the others?
```

Comment:
Question :D

Reply:
which thing doesn't exist?

Reply:
The element we're looking for. For example,  if the data commitment doesn't exist, it will just return `nil` and not throw an exception...

Reply:
ahh, I see. Intuitively it makes sense to throw and error, but I'm not 110% confident in that decision.

Reply:
Will create an issue to think about it later

Reply:
https://github.com/celestiaorg/celestia-app/issues/437